doc/src/conf.py

```diff
@@ -204,5 +207,74 @@
 ]
 
 # Use svg for graphviz
-
 graphviz_output_format = 'svg'
+
+
+# Requried for linkcode extension.
+try:
+    commit_hash = subprocess.check_output(['git', 'rev-parse', 'HEAD'])
+    commit_hash = commit_hash.decode('ascii')
+    commit_hash = commit_hash.rstrip()
+except:
+    import warnings
+    warnings.warn(
+        "Failed to get the git commit hash as the command " \
+        "'git rev-parse HEAD' is not working. The commit hash will be " \
+        "assumed as the SymPy master, but the lines may be misleading " \
+        "or nonexistant as it is not the correct branch the doc is " \
+        "built with. Check your installation of 'git' if you want to " \
+        "resolve this warning.")
```

Comment:
The docs are shipped with the source tarball, so there are legitimate reasons why this would occur.